docs/edit_first_game_server.md

```diff
@@ -56,14 +71,32 @@ Modify the following line from gameserver.yaml to use the new configuration.
         image: gcr.io/[PROJECT_ID]/agones-udp-server:0.2
 ```
 
-### Deploy Server
+### For Minikube local development, modify gameserver.yaml
+Modify the following line from gameserver.yaml to use the new configuration.
+
+```
+    spec:
+      containers:
+      - name: agones-simple-udp
+        image: gcr.io/[PROJECT_ID]/agones-udp-server:0.2
+        imagePullPolicy: Never
```

Comment:
Do we need `imagePullPolicy: Never` ? it is the default.

Reply:
I found I needed it or I would get an error as it failed to pull the image.  I'll try without it when not using make.

Reply:
Hmn. That's weird. Actually, I lie - `Never` is not the default `IfNotPresent` - but if you have transferred the image to Minikube, it shouldn't pull it then.

Reply:
imagePullPolicy: Never has been removed.  Now both GKE and Minikube share the same paragraph.